File: app/src/main/java/com/azure/minitwitter/retrofit/response/TweetDeleted.java
package com.azure.minitwitter.retrofit.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class TweetDeleted {

    @SerializedName("message")
    @Expose
    private String message;

    @SerializedName("user")
    @Expose
    private User user;

    public TweetDeleted() {
    }

    public TweetDeleted(String message, User user) {
        this.message = message;
        this.user = user;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }
}

File: app/src/main/java/com/azure/minitwitter/retrofit/response/TweetDeleted.kt
package com.azure.minitwitter.retrofit.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class TweetDeleted (
        @SerializedName("message") @Expose val message: String,
        @SerializedName("user") @Expose val user: User
)